package.json

```diff
@@ -52,6 +52,9 @@
 			"svelte-preprocess",
 			"swiper",
 			"unrs-resolver"
+		],
+		"onlyBuiltDependencies": [
+			"@parcel/watcher"
```

Comment:
We can probably put this into ignoredBuiltDependencies instead. It looks like their install script doesn't do anything unless a specific environment variable is set.